### ISSUE TITLE
Bootstrap variable file should not be introduced in layout.css

### ISSUE THREAD

**Issue by zhnoah** [ttskch/select2-bootstrap4-theme#43]
If I use gulp-sass instead of sass-loader to compile SCSS source code, I will not be able to proceed.

I think it is better to add in *webpack.config.js*:

```js
{
    loader: 'sass-loader',
    options: {
        prependData: `
            @import "~bootstrap/scss/functions";
            @import "~bootstrap/scss/variables";
            @import "~bootstrap/scss/mixins";
        `
    }
}
```

**Comment by dangelion** [comment]
@zhnoah is totally right, these @import

```
@import "~bootstrap/scss/functions";
@import "~bootstrap/scss/variables";
@import "~bootstrap/scss/mixins";
```

have not to be imported there. Here this causes serious issues when using Bootstrap with RFS enabled.
Please, could you fix it?
Thanks

**Comment by zhnoah** [comment]
@dangelion 

see [https://github.com/ttskch/select2-bootstrap4-theme/pull/44](here)

**Comment by ttskch** [comment]
@zhnoah @dangelion Sorry for my late response. I've just fixed this via e946eed. I'm publishing soon. Thanks.
